### PR TITLE
Delete branches after merge

### PR DESCRIPTION
**Pull request by martin-v**

### CODE REVIEW

File: src/main/java/org/retest/rebazer/connector/GithubConnector.java [hunk 1]
```diff
@@ -70,6 +70,8 @@ public void merge( final PullRequest pullRequest ) {
 		request.put( "merge_method", "merge" );
 
 		template.put( requestPath( pullRequest ) + "/merge", request, Object.class );
+
+		template.delete( "/git/refs/heads/" + pullRequest.getSource() );
```
Comment (diba1013) [inline src/main/java/org/retest/rebazer/connector/GithubConnector.java]: Is this tested?

Comment (diba1013) [inline src/main/java/org/retest/rebazer/connector/GithubConnector.java]: Is merging executed asynchronously? If yes, is deletion executed afterwards or will there be conflicts if deletion is faster?

Comment (martin-v) [reply src/main/java/org/retest/rebazer/connector/GithubConnector.java]: Yes manually

Comment (martin-v) [reply src/main/java/org/retest/rebazer/connector/GithubConnector.java]: It's synchronously

**Pull request merged**